unblob/cli.py

```diff
@@ -39,7 +39,8 @@ def cli(files: Tuple[Path], extract_root: Path, depth: int, verbose: bool):
     configure_logger(verbose, extract_root)
     logger.info("Start processing files", count=len(files))
     for path in files:
-        process_file(path.parent, path, extract_root, max_depth=depth)
+        root = path if path.is_dir() else path.parent
```

Comment:
This would need a more elaborate explanation in the commit message, I would write an example. Also there is a typo in it.